vault/audit_test.go

```diff
@@ -239,7 +239,7 @@ func TestCore_DisableAudit(t *testing.T) {
 	c, keys, _ := TestCoreUnsealed(t)
 	c.auditBackends["noop"] = corehelpers.NoopAuditFactory(nil)
 
-	existed, err := c.disableAudit(namespace.RootContext(nil), "foo", true)
+	existed, err := c.disableAudit(namespace.RootContext(context.TODO()), "foo", true)
```

Comment:
`context.TODO()` was the suggested replacement in the go-staticcheck warning. But I can give `context.Background()` a try as well.

Reply:
`context.TODO()` was the suggested replacement in the go-staticcheck warning. But I can give `context.Background()` a try as well.

Reply:
I've replaced `context.TODO()` with `context.Background()` and all of the tests continue passing, so I'm in favour of using the **Background** function over the **TODO** function, especially since I noticed afterwards, other tests were also using the **Background** function when a **Context** was needed.